data/coalition/coalition culture conversations.txt

```diff
@@ -233,3 +233,137 @@ mission "Coalition Folklore: Kimek"
+			`	Within a few minutes that must've felt like hours for the stationed Heliarch agents, battalions of drones start making their way from the tallest building in the city - the hunger tower. Ferrying around several meals each, with impressive precision, the drones fly throughout the civilian mass, distributing the meals. Hundreds of larger head to the residential areas of the city, presumably to provide the necessary supplies for the citizens until the next food convoy arrives. Seemingly satisfied, for the moment at least, the Kimek gradually dissipate out of the spaceport and back to their homes, leaving the spaceport to the now exhausted Heliarch agents.`
+				decline
+
+
+
+mission "Coalition Incident: Arach Store"
+	minor
+	invisible
+	to offer
+		has "Coalition: First Contact: done"
+		random < 2
+	source
+		government "Coalition"
+		attributes "arach"
+		not attributes "coalition station"
+		not planet "Ahr"
+	on offer
+		conversation
+			`A Heliarch patrol seems to be discussing with an Arach shop worker in front of a large, prestigious looking Arach clothing store in the spaceport. Other agents prevent you and other bystanders from approaching the scene, but you can still make out some context from their gesturing. It seems the Heliarchs are there to confiscate some pieces from the shop that you guess don't abide by their laws, while the shop worker keeps worryingly attempting to answer what questions they pose him.`
+			`	Eventually, the shop owner arrives. Judging by her fine clothes, and from the reactions of the local Arachi, you figure she must either be really famous here on <origin>, or possibly belong to one of the Arach Houses, though nothing on her attire seems to indicate any specific house. She tranquilizes the worker, and takes on to speaking with the Heliarch agents as he heads back inside.`
+			`	She tries to diffuse the situation, speaking calmly and attempting to take the items back into the shop, but the Heliarch agents do not budge. Running out of patience, the Heliarch in charge of the patrol cuts the Arach lady off as she's speaking, forcibly taking all the items in question, and having the agents rip apart what you guess must've been the offending parts of the cloth pieces. Stunned, the Arach simply watches as they render most of the items into unusable rags, keeping the bits they hacked away, and handing her back the other tattered remains. Without another word, they leave the scene. The Arach heads on inside, and closes the store.`
+				decline
+
+
+
+mission "Coalition Incident: Ring of Wisdom"
+	minor
+	invisible
+	to offer
+		has "Coalition: First Contact: done"
+		random < 1
```

Comment:
Cant we get it higher since it only offers on the ring of wisdom? (Say 2-4%, maybe?)
Or mb offer it at the other rings, too?

Reply:
Also the Ring of Wisdom is the one world in Coalition space the player will visit the most before choosing a side in the conflict, so even 1% is still likely to be seen by players.